filter_plugins/openshift_master.py

```diff
@@ -237,7 +237,11 @@ def __init__(self, api_version, idp):
         self._required += [['headers']]
         self._optional += [['challengeURL', 'challenge_url'],
                            ['loginURL', 'login_url'],
-                           ['clientCA', 'client_ca']]
+                           ['clientCA', 'client_ca'],
+                           ['clientCommonNames','client_common_names'],
+                           ['emailHeaders','email_headers'],
+                           ['nameHeaders','name_headers'],
+                           ['preferredUsernameHeaders','preferred_username_headers']]
```

Comment:
Pylint is tripping on no whitespace in between these latter options otherwise LGTM.